pkg/ecctl/init.go

```diff
@@ -287,6 +355,73 @@ func askInfraSelection(cfg *Config, scanner *input.Scanner, writer, errWriter io
 	return nil
 }
 
+//nolint
+func askRegionSelection(cfg *Config, scanner *input.Scanner, writer io.Writer) error {
+	regionChoiceRaw := scanner.Scan(regionChoiceMsg)
+	fmt.Fprintln(writer)
+	regionChoice, err := strconv.Atoi(regionChoiceRaw)
+	if err != nil {
+		return err
+	}
+
+	switch regionChoice {
```

Comment:
We can replace all of this for `map[int]string` and then having a `region, ok := regionMap[regionChoice]` Handling the `ok` as the error and returning the region, It'll make the code shorter.